src/components/Clickable.js

```diff
@@ -0,0 +1,23 @@
+import React, {Component} from 'react'
+import Button from '@material-ui/core/Button'
```

Comment:
Include all the other files in your src, public, and root directories. excluding the node_modules folder.  So that anyone who checks out the repository will be able to build/launch

Reply:
I think I got all of the needed files included now!